client-v2/src/util/form.ts

```diff
@@ -101,14 +105,30 @@ export const getInitialValuesForArticleFragmentForm = (
           src: article.imageCutoutSrc,
           width: strToInt(article.imageCutoutSrcWidth),
           height: strToInt(article.imageCutoutSrcHeight),
-          origin: article.imageCutoutSrcOrigin
+          origin: article.imageCutoutSrcOrigin,
+          thumb: article.imageCutoutSrc
```

Comment:
@akash1810 This thumbnail is the original image -- as far as I can see from the modelling and old tool, there isn't a thumbnail available in the article meta and the old tool simply uses the entire image here, but I thought I'd run this by you!